### PR TITLE
New translation system for PrestaShop 1.7

### PR DESCRIPTION
**Pull request by AlexEven**

### CODE REVIEW

**Comment by xBorderie** [comment]
6d0e240dc6b93699078941f4a2769ce345dfa392b8ef05154625594940150e6d

File: news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+2. The **Demo** section has only two sections:
+  * Catalog: content for demo product and categories (descriptions).
+  * Pages: content inside the demo pages (formerly known as CMS pages).
+
+3. **Notifications** are split according to their level of alert: error, warning or success. They are the messages showing up on your shop.
+
+4. As for the **Form** sections, you’ll have
+  * the form labels,
+  * its corresponding errors (distinct from the theme errors, this one are specific to the form and will display inline),
```
Comment (llandrysg) [inline news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown:199]: distinct from the theme errors, these one are specific to the form and will display inline

File: news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+
+###  Admin
+Now, let’s see how things are organized for the strings from the back office.
+
+| Domain | Strings |
+| ---------- | -------- |
+| Global | Anything which doesn’t fall in the below categories, but is still related to the back office, that can be found in a lot of occurrences (“Max”, “Settings”, “Enabled”, etc.) AND in different parts of the software too. <br/>If a string is specific to a given page, then it shouldn’t be in the global domain. |
+| Actions | All the call-to-actions, buttons or links that you find on the back office, and that are quite generic (“Save”, “Add”, “Delete”, etc.). Again, if it applies to one page and one functional domain only, then this is not the domain to use. |
+| Notifications | All the warning, error or success messages that can appear in the back office.They must be general notifications applying to any part of the software (e.g. “Settings updated.”).<br/> In case you have a specific notification (“Friendly URLs are currently disabled”), then it should fall under the notification subdomain for its functional domain (here *Admin.Catalog.Notification*). |
```
Comment (llandrysg) [inline news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown:212]: “Settings updated”

File: news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+With the following domains being used:
+
+<img style="border: 1px solid #CCC; padding: 10px;" width="250" src="/assets/images/2016/06/domains_example3.png">
+
+
+
+## Limitations
+While we wanted to have a great system, there are nevertheless a few drawbacks.
+
+Where we used to have a string with a feature, and the following string the help text attached to it; they will now be in two different files.
```
Comment (llandrysg) [inline news/_posts/translation/2016-07-01-new-translation-system-prestashop-17.markdown:247]: +Where we used to have a string with a feature, and in the following string the help text attached to it; they will now be in two different files.

**Pull request merged**